packages/apollo-codegen-swift/src/__tests__/__snapshots__/codeGeneration.ts.snap

```diff
@@ -108,7 +108,7 @@ exports[`Swift code generation #classDeclarationForOperation() should correctly
   ///   }
   /// }
   public let operationDefinition =
-    \\"mutation CreateReview($episode: Episode) {\\\\n  createReview(episode: $episode, review: {stars: 5, commentary: \\\\\\"\\\\\\"\\\\\\"\\\\n    Wow!\\\\n     I thought\\\\n      This movie \\\\ ROCKED!\\\\n  \\\\\\"\\\\\\"\\\\\\"}) {\\\\n    stars\\\\n    commentary\\\\n  }\\\\n}\\"
+    \\"mutation CreateReview($episode: Episode) {\\\\n  createReview(episode: $episode, review: {stars: 5, commentary: \\\\\\"\\\\\\"\\\\\\"\\\\n    Wow!\\\\n     I thought\\\\n      This movie \\\\\\\\ ROCKED!\\\\n  \\\\\\"\\\\\\"\\\\\\"}) {\\\\n    stars\\\\n    commentary\\\\n  }\\\\n}\\"
```

Comment:
I'm not familiar with snap but it looks like everything here is double-escaped, right? So the old output ended up evaluating to `This movie \ ROCKED!` which is of course bogus because that suggests we're escaping a space.

That was actually how the old code worked since our `escapedString` function wasn't escaping backslashes, it was only escaping quotes and converting newlines into `\n`. So a backslash in the input string would stay as an unescaped backslash in the output string, and therefore be interpreted as an escape by Swift (such that the backslash isn't present in the runtime value of the string).

The new version of `escapedString` actually escapes everything that's supposed to be escaped, so the string literal now reads `This movie \\ ROCKED!`, and therefore the backslash is preserved in the runtime string value.

Reply:
Yeah this stuff with the "It needs to be escaped exactly double the number of times" thing is super-confusing. Thanks for the explanation!